couch/tests/compose/compose_v2.yaml

```diff
@@ -2,7 +2,7 @@ version: '3.5'
 
 services:
   couch:
-    image: klaemo/couchdb:${COUCH_VERSION}
+    image: datadog/docker-library:couch_2_3
```

Comment:
We still need to test 1.6 🙂

Reply:
1.6 doesn't use the same compose file?

Reply:
Ah, good. We should still use the env var here though

Reply:
Fair, done.